.github/workflows/sonarqube.yml

```diff
@@ -9,11 +9,11 @@ jobs:
     runs-on: ubuntu-20.04
     env:
         TZ: Asia/Kolkata
-        SONAR_ORGANIZATION: ${{ secrets.SONAR_ORGANIZATION }}
-        SONAR_PROJECT_KEY: ${{ secrets.SONAR_PROJECT_KEY }}
-        SONAR_HOST_URL: ${{ secrets.SONAR_HOST_URL }}
-        SONAR_TOKEN: ${{ secrets.SONAR_TOKEN }}
+        SONAR_ORGANIZATION: apache
```

Comment:
Petri, do you think we could extract these env variables into GH secrets?

The reason it would be really useful is because there are forks out there which gets regularly synced from this repo and all those forks could also utilize their own sonar instance for checks just by reconfiguring their secrets instead of changing the workflow code.

What do you think?

Reply:
Hi - we don't have the rights to add secrets to our repo as it is owned by Apache. @vorburger worked out the process to get secrets added using the REST API - you can find the description here (last paragraph): http://blog2.vorburger.ch/2020/05/fineractdev-cicd-from-github-to-google.html?m=1

I haven't tried this, but if you want to give it a go - then by all means go for it...

Reply:
Sounds good, let me do it and get back to you.

Reply:
Except it might now be even easier at it seems they've added secrets support to the gh cli: https://github.com/cli/cli/releases/tag/v1.4.0

Reply:
Yep, just used the CLI for it, here are the commands I did:
```
$ gh secret set SONAR_PROJECT_KEY -b "apache"
✓ Set Actions secret SONAR_PROJECT_KEY for apache/fineract
$ gh secret set SONAR_PROJECT_KEY -b "apache_fineract"
✓ Set Actions secret SONAR_PROJECT_KEY for apache/fineract
$ gh secret set SONAR_HOST_URL -b "https://sonarcloud.io"
✓ Set Actions secret SONAR_HOST_URL for apache/fineract
```

Reply:
Do you mind switching these back to secrets so we can try it? Hopefully that should be it and we can get rid of the travis build completely.

Reply:
Ok - done. Seems to work - can you have a look and merge.